--- local.conf  2024-11-18 16:50:26.789406202 +0000
+++ local.conf  2024-11-18 16:51:22.376822973 +0000
@@ -29,7 +29,9 @@
 # There are also the following hardware board target machines included for 
 # demonstration purposes:
 #
-#MACHINE ?= "beaglebone-yocto"
+MACHINE ?= "beaglebone-yocto"
+RM_OLD_IMAGE = "1"
+INHERIT += "rm_work"
 #MACHINE ?= "genericx86"
 #MACHINE ?= "genericx86-64"
 #MACHINE ?= "edgerouter"
